Python/Product/TestAdapter.Executor/Pytest/PytestExtensions.cs

```diff
@@ -64,13 +64,13 @@ string projectHome
             var pytestId = CreateProperCasedPytestId(sourceFullPath, projectHome, test.Id);
             var fullyQualifiedName = CreateFullyQualifiedTestNameFromId(sourceFullPath, pytestId);
             var tc = new TestCase(fullyQualifiedName, PythonConstants.PytestExecutorUri, sourceFullPath) {
-                DisplayName = test.Name,
+                DisplayName = FixupParameterSets(test.Name),
                 LineNumber = line,
                 CodeFilePath = sourceFullPath
             };
 
             tc.SetPropertyValue(Constants.PytestIdProperty, pytestId);
```

Comment:
Note we dont modify the actual pytestId.. that is the string that execution passes on command line to pytest.  this is just a fix for Test Explorer